back-office/shop-parameters/customer-settings/groups.md

```diff
@@ -0,0 +1,141 @@
+
+Only products that have the selected category as their default category should be affected by the discount. The expected behavior of a category discount works as followed: it prevails over any discount a member of this group could benefit from in this category. See notifications below.
+
+An information notification id displayed: `Caution: The discount applied to a category does not stack with the overall reduction but instead replaces it.` in `Admin.Shopparameters.Notification`. Right below is a warning notification displayed: `Only products that have this category as the default category will be affected.` in `Admin.Shopparameters.Notification`.
+
+**Discount (%).** Only numbers can be typed in the field. If the merchant tries to save letters or a negative value, then he should stay on this modal window and get an error notification right below the field: `The discount value is incorrect (must be a percentage).` in `Admin.Shopparameters.Notification`.
+
+Clicking 'Add' saves preferences. It displays the category discount(s) in the 'Edit' form, just below the 'Category discount' field, with the following information: route to the category, discount value, and a 'Delete' button. If a user deletes a discount, the line simply disappears.
+
+
+### Authorized modules
+
+With this setting, **merchants can authorize or prevent members of this group to access and use some of the shop's modules**. A list of all the store's enabled modules is displayed with a switch button to enable or disable each module. By default, all modules are enabled.
+
+Two actions are available at the end of the page: _Save_ and _Cancel_. When saving, the user gets redirected to the Shop Parameters > Customer Settings > Groups page with a success notification: 'Successful update.'.
+
+
+## View form
+
+A summary of the preferences set for a customers group is accessible through the 'View' action or just by clicking on the line in the groups listing. It opens a new page divided into three parts:
+
+### Group information
+
+Merchants can find all settings in this part:
+
+- Name of the group of customers<br>
+- Discount<br>
+- Price display method<br>
+- Show prices
+
+
+### Current category discount
+
+A warning notification, _None_, is displayed by default if the customers group has no category discount. If there is one or several, it takes the form of a listing displaying the following information:
+
+- Category<br>
+- Discount
+
+ 
+### Members of this customer group
+
+A wording is supposed to explicit how group members are displayed: `Limited to the first 100 customers.` and `Please use filters to narrow your search.` in `Admin.Shopparameters.Feature`.
+
+By default, the table is divided into the following columns:
+
+- ID<br>
+- Social title<br>
+- First names<br>
+- Last name<br>
+- Email address<br>
+- Date of birth<br>
+- Registration date<br>
+- Enabled
```

Comment:
It seems we can't change the status of the customer

Reply:
Yes, just found the issue: https://github.com/PrestaShop/PrestaShop/issues/21847 :sweat: